homeassistant/components/refoss/sensor.py

```diff
@@ -104,6 +104,66 @@ class RefossSensorEntityDescription(SensorEntityDescription):
             fn=lambda x: abs(x) if x < 0 else 0,
         ),
     ),
+    "em16": (
```

Comment:
I am now looking at this, and this list is exactly the same as the `em06` one. I'd rather see that deduplicated